Assets/Mariana /Script/SceneLoadManager.cs

```diff
@@ -0,0 +1,15 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class SceneLoadManager : MonoBehaviour
+{
+    // Start is called before the first frame update
+    public void PlayGame()
+    {
+        SceneManager.LoadScene(SceneManager.GetActiveScene().buildIndex + 1);
```

Comment:
Unfortunately, this is going to be unreliable long term since the scene index will change over the life of the project.